conf/production.ini

```diff
@@ -65,6 +65,9 @@ webassets.base_dir: h:static
 webassets.base_url: assets
 webassets.cache_max_age: 31536000
 webassets.static_view: True
+webassets.uglifyjs_bin: node_modules/.bin/uglifyjs
```

Comment:
I took these out because the herokuish stack configured the $PATH sensible. Can we do the same elsewhere? I think production.ini shouldn't be hard coding this.